Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -77,10 +81,34 @@ public class BellerophonManager: NSObject {
             }
         }
     }
-
+    
+    /// Updates the display with the given status.
+    ///
+    /// - Parameter status: Kill switch status to check if the kill switch view should be displayed.
+    @objc public func updateDisplay(_ status: BellerophonObservable) {
```

Comment:
can we have tests on this function to make sure we are calling the correct functions and we get the expected behavior?

Reply:
Tests added